### PR TITLE
Consumer polling

### PR DESCRIPTION
**Pull request by sylfabre**

### CODE REVIEW

File: src/Impl/ConsumerPollingTrait.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+     * @var integer
+     */
+    protected $pollingInterval = 1000;
+
+    /**
+     * Set polling interval in milliseconds.
+     */
+    public function setPollingInterval(int $msec): void
+    {
+        $this->pollingInterval = $msec;
```
Comment (makasim) [inline src/Impl/ConsumerPollingTrait.php]: should return `$this` and return typehint `: self`

Comment (makasim) [reply src/Impl/ConsumerPollingTrait.php]: @sylfabre ping

Comment (makasim) [reply src/Impl/ConsumerPollingTrait.php]: @sylfabre ping

Comment (sylfabre) [reply src/Impl/ConsumerPollingTrait.php]: @makasim forgot to push on Friday!

Comment (sylfabre) [reply src/Impl/ConsumerPollingTrait.php]: @makasim forgot to push on Friday!

Comment (sylfabre) [reply src/Impl/ConsumerPollingTrait.php]: @makasim forgot to push on Friday!

Comment (sylfabre) [reply src/Impl/ConsumerPollingTrait.php]: @makasim forgot to push on Friday!

Comment (sylfabre) [reply src/Impl/ConsumerPollingTrait.php]: @makasim forgot to push on Friday!

Comment (makasim) [reply src/Impl/ConsumerPollingTrait.php]: @sylfabre

Comment (makasim) [reply src/Impl/ConsumerPollingTrait.php]: @sylfabre ping

**Pull request merged**